packages/zwave-js/src/lib/controller/Controller.ts

```diff
@@ -1901,7 +1906,7 @@ ${associatedNodes.join(", ")}`,
 	public async replaceFailedNode(
 		nodeId: number,
 		includeNonSecure: boolean = false,
-	): Promise<boolean> {
```

Comment:
Hm, maybe we should avoid starting this process twice (just like inclusion/exclusion)

Reply:
I could use the inclusionActive flag?